src/tensorlake/http_client.py

```diff
@@ -161,7 +174,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, traceback):
-        self.close()
+        self._close()
```

Comment:
I'm wondering if this was the source of leaking socket warning messages when we are running SDK tests

Reply:
I still see tons of them unfortunately.